experimental/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import { EventEmitter } from "events";
+import { IAudience } from "@fluidframework/container-definitions";
+import { Container } from "@fluidframework/container-loader";
+import { RootDataObject } from "./rootDataObject";
+
+export class FluidContainer
+    extends EventEmitter
+    implements
+        Pick<Container, "audience" | "clientId" | "close" | "closed" | "connected" >,
+        Pick<RootDataObject, "initialObjects"> {
+    private readonly _container: Container | undefined = undefined;
+    private readonly _rootDataObject: RootDataObject | undefined = undefined;
+
+    public constructor(container: Container, dataObject: RootDataObject) {
```

Comment:
```suggestion
    public constructor(private readonly _container: Container, private readonly _rootDataObject: RootDataObject) {
```
This would allow you to not have to declare them in line 16-17 as `undefined` and remove the need for the undefined checks in the getters below

Reply:
If you need a new verb you could use `load`, which is what a lot of other classes in our codebase use?

Reply:
i'm good with load. not a huge fan of having the same 'verb' do two different things based on if it's static or part of the returned class

Reply:
the private readonly on the constructor gives me a TS error

![image](https://user-images.githubusercontent.com/1434956/118156507-48be5480-b3ce-11eb-878e-fceaa73b98b1.png)

Reply:
Is that image the `private constructor` error?  That one looks like because you added `rootDataObject` but didn't replace usages of `_rootDataObject`.  Sumedh's suggestion from earlier also means you don't need to have them declared in the class and assigned in the constructor body, because that's happening inline here

Reply:
Oh i gotcha. I didn't know that the class had direct access to those constructor params. Lesson learned!